features/rake_task/rake_task.feature

```diff
@@ -93,4 +94,7 @@ Feature: Reek can be driven through its Task
     Then it succeeds
     And it reports:
       """
+      [DEPRECATED]
+      Starting with reek 1.6 there is only one configuration file possible.
+      Please use "config_file" instead of "config_files".
```

Comment:
At one point in time we should get rid of "config_files" so let's start by deprecating it and remove it in a year or so.